app/views/partners/PagePartnerResetPassword/index.vue

```diff
@@ -0,0 +1,154 @@
+<script>
+import ChangePasswordModal from 'app/views/user/ChangePasswordModal.vue'
+import { mapActions } from 'vuex'
+import apiClient from 'app/core/api/api-clients'
+
+export default Vue.extend({
+  components: {
+    ChangePasswordModal,
+  },
+  data () {
+    return {
+      userInput: '',
+      foundUsers: [],
+      showChangePasswordModal: false,
+      selectedUser: '',
+    }
+  },
+  computed: {
+    isAPIClient () {
+      return me.isAPIClient() || me.isAdmin()
+    },
+  },
+  async created () {
+    this.clientId = await this.fetchClientId()
```

Comment:
You don't need to fetch the `clientId`, you'll [already have it on backend side](https://github.com/codecombat/codecombat-server/pull/1118#discussion_r1863556129) when sending the getUsersSearch request.